aws/scenarios/microVMs/microvms/resources/resources.go

```diff
@@ -69,8 +72,38 @@ func formatResourceXML(xml string, args map[string]pulumi.StringInput) pulumi.St
 	return pulumiXML
 }
 
+func isLocalRecipe(recipe string) bool {
+	return (recipe == CustomLocalRecipe) || (recipe == DistroLocalRecipe)
+}
+
+func getArchSpecificRecipe(recipe string) string {
+	var prefix string
+
+	if !isLocalRecipe(recipe) {
+		return recipe
+	}
+
+	if strings.HasPrefix(recipe, "distro") {
+		prefix = "distro"
+	} else if strings.HasPrefix(recipe, "custom") {
+		prefix = "custom"
+	} else {
+		panic("unknown recipe " + recipe)
+	}
+
+	if runtime.GOARCH == "amd64" {
+		return fmt.Sprintf("%s-amd64", prefix)
+	} else if runtime.GOARCH == "arm64" {
+		return fmt.Sprintf("%s-arm64", prefix)
+	}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
This could be `getLocalArchRecipe`

Reply:
I meant only the selected part, could be exported to `getLocalArchRecipe` not urgent and not a blocker ! Will let your team review this code, as you have ownership on it